serving/accessing-logs.md

```diff
@@ -1,110 +1,102 @@
-```
-
-This command starts a local proxy of Kibana on port 8001. For security reasons, the 
-Kibana UI is exposed only within the cluster.
+  This command starts a local proxy of Kibana on port 8001. For security reasons, the
+  Kibana UI is exposed only within the cluster.
 
-Navigate to the
-[Kibana UI](http://localhost:8001/api/v1/namespaces/monitoring/services/kibana-logging/proxy/app/kibana)
- (*It might take a couple of minutes for the proxy to work*).
+* Navigate to the
+[Kibana UI](http://localhost:8001/api/v1/namespaces/monitoring/services/kibana-logging/proxy/app/kibana). *It might take a couple of minutes for the proxy to work*.
 
-The Discover tab of the Kibana UI looks like this:
+  The Discover tab of the Kibana UI looks like this:
 
-![Kibana UI Discover tab](./images/kibana-discover-tab-annotated.png)
+  ![Kibana UI Discover tab](./images/kibana-discover-tab-annotated.png)
 
-You can change the time frame of logs Kibana displays in the upper right corner
-of the screen. The main search bar is across the top of the Discover page.
+  You can change the time frame of logs Kibana displays in the upper right corner
+  of the screen. The main search bar is across the top of the Discover page.
 
-As more logs are ingested, new fields will be discovered. To have them indexed,
+* As more logs are ingested, new fields will be discovered. To have them indexed,
 go to Management > Index Patterns > Refresh button (on top right) > Refresh
 fields.
 
 <!-- TODO: create a video walkthrough of the Kibana UI -->
 
 ### Accessing configuration and revision logs
 
-To access the logs for a configuration, enter the following search query in Kibana:
-
-```text
+* To access the logs for a configuration, enter the following search query in Kibana:
+```
 kubernetes.labels.serving_knative_dev\/configuration: "configuration-example"
 ```
 
-Replace `configuration-example` with your configuration's name. Enter the following
+* Replace `configuration-example` with your configuration's name. Enter the following
 command to get your configuration's name:
-
-```shell
+```
 kubectl get configurations
 ```
 
-To access logs for a revision, enter the following search query in Kibana:
-
-```text
+* To access logs for a revision, enter the following search query in Kibana:
+```
 kubernetes.labels.serving_knative_dev\/revision: "configuration-example-00001"
 ```
 
-Replace `configuration-example-00001` with your revision's name.
+  Replace `configuration-example-00001` with your revision's name.
 
 ### Accessing build logs
 
-To access the logs for a build, enter the following search query in Kibana:
-
-```text
+* To access the logs for a build, enter the following search query in Kibana:
+```
 kubernetes.labels.build\-name: "test-build"
 ```
 
-Replace `test-build` with your build's name. The build name is specified in the `.yaml` file as follows:
+  Replace `test-build` with your build's name. The build name is specified in the `.yaml` file as follows:
```

Comment:
Many users may also use `generateName` with `kubectl create`. It might also be worth pointing out `kubectl get builds`.